ci/kind/kind-setup.sh

```diff
@@ -143,10 +143,8 @@ function configure_networks {
         echo "disconnected worker $rn from network $n"
       fi
     done
-    if [[ $n != "kind" ]]; then
```

Comment:
I thought about it some more, and this change is not correct. We should keep this test, and just use `CLUSTER_NAME` for the network name.